plugin.d.ts

```diff
@@ -6,7 +6,8 @@ declare namespace fastifyBearerAuth {
 		auth?: (key: string, req: fastify.FastifyRequest) => boolean | Promise<boolean>,
 		errorResponse?: (err: Error) => { error: string },
 		contentType?: string,
-		bearerType?: string
+		bearerType?: string,
+		allowAnonymous?: true
 	}
```

Comment:
```suggestion
	}
	
	interface FastifyContext {
		config: {
			allowAnonymous?: boolean
		}
	}
```

You missed the route context typing.

Reply:
Thanks